### ISSUE TITLE
Værn om højtiderne!

### ISSUE THREAD

**Issue by sshine** [Athas/EggsML#127]
- [ ] Sørg for at royale fødselsdage er tidssvarende. `eggspi/royalsToday` henter royale fødselsdage fra `$CONCIEGGS_DB_DIR/royale-fødselsdage/royale_fødselsdage`, som er et tilfældigt udsnit af flagdage fra 2017 (så vidt jeg husker Niels sige).
- [ ] Tilføj en mekanisme til at afgøre om det er påske.
- [x] Julen er vist allerede håndteret. Lad os vente til jul med at gøre den mere julet.
- [ ] Tilføj helligdage som blev afskaffet af [Helligdagsreformen af 1770](https://da.wikipedia.org/wiki/Helligdagsreformen_af_1770): Fx 3. juledag, Helligtrekongersdag, Sankt Hansdag, Allehelgensdag, og aller vigtigst bør vi være opmærksomme på at *Taksigelsesdag for afslutningen på [Københavns brand 1728](https://da.wikipedia.org/wiki/K%C3%B8benhavns_brand_1728)* blev flyttet til første søndag efter 23. oktober.
- [ ] Tilføj fjollede højtider såsom skærtirsdag, 157. påskedag, 19. søndag i advent

En fælles grænseflade for funktioner der fortæller om en givet dato er en særlig højtid kunne være: `eggspi/<kommando> [yyyy-mm-dd]`, hvorefter kommandoen kan printe en linje for hver højtid den genkender. Så kan de samles under én fane, som kan vælge en tilfældig grund til at fejre en givet dag blandt alle dens højtidsleverandørkommandoer.

**Comment by svip** [comment]
Vent.  Hvis 19. søndag i advent er en ting, så er alle søndage da vel en søndag i advent?

Jeg støtter i øvrigt fuldt op om dette emne!  Faktisk syntes jeg vi skal starte et borgerforslag om at afskaffe Helligdagsreformen af 1770!

**Comment by sshine** [comment]
@svip: Ja, hvis ikke søndagen udmærker sig af nogen bedre grund.

**Comment by svip** [comment]
Men én dag kan jo godt være mere end én helligdag.  Påskesøndag kan jo også være en søndag i advent.  Det er vel vigtigt at dette program understøtter flere helligdage på samme dag!

**Comment by sshine** [comment]
@svip: Ja, det er derfor jeg foreslår at funktioner som foreslår højtider (om det er royale fødselsdage, Hitlers fødselsdag, Danmarks befrielsesdag, påskedagene, eller hvad det nu måtte være, der er værd at sige om en givet dato), giver ét resultat per linje. Gerne prioriteret efter vigtighed, egentlig. Så kan en eventuel aftager af resultaterne selv vælge en tilfældig højtid eller hvad der nu måtte være formålet.

**Comment by nqpz** [comment]
Jeg er helt enig i at det vil være godt at beskrive hellidagsmønstrene i et ikke-Turing-komplet deklarativt sprog. Det er simpelthen bare pænere.
